### ISSUE TITLE
Tor support not working

### ISSUE THREAD

**Issue by betsythefc** [ZeeRooo/MaterialFBook#26]
When I enable Orbot proxy in the settings, the page doesn't load using tor. I can turn Orbot off and facebook will still load in the app. This also means that the loading through tor option is not using facebookcorewwwi.onion url.

**Comment by ZeeRooo** [comment]
No, because the onion url is for facebook in desktop version. It only
change thw proxy

**Comment by betsythefc** [comment]
This address works for mobile: https://m.facebookcorewwwi.onion/

**Comment by betsythefc** [comment]
Also the proxy isn't working, as I have option selected and orbital disconnected and facebook still loads in the app

**Comment by ZeeRooo** [comment]
Ok, let me try the mobile version
